core/adapters/http_test.go

```diff
@@ -653,3 +654,17 @@ func TestHTTP_BuildingURL(t *testing.T) {
 		})
 	}
 }
+
+func Test_HTTP_ParseTimeout(t *testing.T) {
+	t.Parallel()
+
+	defaultTimeout, _ := time.ParseDuration("5s")
```

Comment:
What about using table driven tests instead?  Its the de-facto in the golang world, and it fits nicely in this context.

https://github.com/smartcontractkit/chainlink/blob/develop/core/utils/big_test.go#L177-L192